frontend/packages/ceph-storage-plugin/src/plugin.ts

```diff
@@ -180,6 +185,24 @@ const plugin: Plugin<ConsumedExtensions> = [
       },
     },
   },
+  {
+    type: 'Page/Route',
+    properties: {
+      exact: true,
+      path: `/k8s/ns/:ns/persistentvolumeclaims/:appName/volumesnapshots/:name`,
```

Comment:
Please follow the patterns for other pages. Then you can just register this using the model.

Reply:
@bipuladh @a2batic The `Page/Route` extension is very powerful, capable of adding new pages or overriding existing ones.

If I understand correctly, we basically want to provide list & details pages for `VolumeSnapshot` CRD. If so, please use the following extensions:

- `Page/Resource/List`
- `Page/Resource/Details`

to register the corresponding list & details pages.